src/Glpi/Form/QuestionType/QuestionTypeItem.php

```diff
@@ -0,0 +1,261 @@
+            ),
+            __('Assistance') => array_merge(
+                $CFG_GLPI['itil_types'],
+                [
+                    TicketRecurrent::class,
+                ]
+            ),
+            __('Management') => $CFG_GLPI['management_types'],
+            __('Tools') => $CFG_GLPI['tools_types'],
+            __('Administration') => $CFG_GLPI['admin_types']
+        ];
+    }
+
+    /**
+     * Retrieve the default value for the item question type
+     *
+     * @param Question|null $question The question to retrieve the default value from
+     * @return ?string
+     */
+    public function getDefaultValueItemtype(?Question $question): ?string
+    {
+        if (
+            $question !== null
+            && isset($question->fields['default_value'])
+        ) {
+            return json_decode($question->fields['default_value'], true)['itemtype'] ?? null;
+        }
+
+        return null;
+    }
+
+    /**
+     * Retrieve the default value for the item question type
+     *
+     * @param Question|null $question The question to retrieve the default value from
+     * @return int
+     */
+    public function getDefaultValueItemId(?Question $question): int
+    {
+        if (
+            $question !== null
+            && isset($question->fields['default_value'])
+        ) {
+            return json_decode($question->fields['default_value'], true)['items_id'] ?? 0;
+        }
+
+        return 0;
+    }
+
+    #[Override]
+    public function loadJavascriptFiles(): array
+    {
+        return ['js/form_question_item.js'];
+    }
+
+    #[Override]
+    public function formatDefaultValueForDB(mixed $value): ?string
+    {
+        if (is_array($value)) {
+            return json_encode($value);
+        }
+
+        return null;
+    }
+
+    #[Override]
+    public function renderAdministrationTemplate(?Question $question): string
+    {
+        $template = <<<TWIG
+            {% import 'components/form/fields_macros.html.twig' as fields %}
+
+            {% set rand = random() %}
+
+            {{ fields.dropdownItemsFromItemtypes(
+                'default_value',
+                '',
+                {
+                    'init'               : init,
+                    'itemtypes'          : itemtypes,
+                    'no_label'           : true,
+                    'display_emptychoice': true,
+                    'default_itemtype'   : default_itemtype,
+                    'default_items_id'   : default_items_id,
+                    'itemtype_name'      : 'default_value[itemtype]',
+                    'items_id_name'      : 'default_value[items_id]',
+                    'width'              : '100%',
+                    'container_css_class': 'mt-2',
+                    'no_sort'            : true,
+                    'aria_label'         : itemtype_aria_label,
+                    'specific_tags'      : {
+                        'aria-label': items_id_aria_label,
+                    },
```

Comment:
I suppose the first `aria_label` is applied to the main dropdown, while the `specific_tags` are applied to the second dropdown ?

Someone might try to use `specific_tags` to add tags to the main dropdown and wonder why it doesn't work.
I guess we need the option name to indicate that is applies to the second dropdown (sadly existing options are not well named so we have no example to copy).